packages/esm-patient-immunizations-app/src/immunizations/immunization-history-card.scss

```diff
@@ -0,0 +1,70 @@
+@use '@carbon/type';
+@use '@openmrs/esm-styleguide/src/vars' as *;
+
+.widgetCard {
+  border: 1px solid $ui-03;
+}
+
+.content {
+  background-color: $ui-background;
+}
+
+.doseCell {
+  min-width: 140px;
+  padding: 0.5rem;
+  display: inline-block;
+  text-align: center;
+}
+
+.doseLabel {
+  margin-bottom: 0.5rem;
+}
+
+.doseDate {
+  font-size: 0.875rem;
+  color: $text-02;
+}
+
+.headerRow {
+  position: sticky;
+  top: 0;
+  background-color: #f4f4f4;
+  z-index: 2;
+  border-bottom: 1px solid #ddd;
+}
+
+.headerCell {
+  min-width: 200px;
```

Comment:
Please don't use sizes in `px`. Switch to rem, and ideally, try to stick to either [the Carbon spacing tokens](https://carbondesignsystem.com/elements/spacing/overview/) or numbers along those lines (e.g., 12rem == 192px ~= 200px).